packages/account/src/Components/file-uploader-container/file-uploader.tsx

```diff
@@ -13,28 +13,27 @@ type TFileUploaderProps = {
     onError?: (error_message: string) => void;
 };
 
-const UploadMessage = observer(() => {
-    const {
-        ui: { is_mobile },
-    } = useStore();
+const UploadMessage = () => {
+    const { isMobile } = useDevice();
+
     return (
         <React.Fragment>
             <Icon icon='IcUpload' className='dc-file-dropzone__message-icon' size={30} />
             <div className='dc-file-dropzone__message-subtitle'>
                 <Text size='xxs' align='center' weight='bold' color='less-prominent'>
-                    {is_mobile ? (
+                    {isMobile ? (
```

Comment:
should we allow drag and drop files in tablet?

Reply:
Let me check on this 😬

Reply:
I have updated the condition.